playbooks/robusta_playbooks/node_actions.py

```diff
@@ -0,0 +1,76 @@
+from typing import List
+
+from kubernetes import client
+from kubernetes.client import V1OwnerReference, V1PodList
+
+from robusta.api import ActionException, ErrorCodes, MarkdownBlock, NodeEvent, action
+
+
+@action
+def cordon(event: NodeEvent):
+    """
+    Cordon a node
+    """
+    node = event.get_node()
```

Comment:
do we have a way of detecting if a node is cordoned/unschedulable from the frontend? If not than i could see using these functions causing undetectable issues from the robusta UI

Reply:
Yes visible, it will have a node.kubernetes.io/unschedulable=None:NoSchedule taint